spec/ttml1.xml

```diff
@@ -4333,52 +4333,82 @@ as a style property only to those element types indicated in the following table
+<code>tts:fontSize="1c"</code> is specified on <code>P</code>, then the computed value of <code>tts:fontSize</code>
+resolves to (36px,36px), while if <code>tts:fontSize="1c 1c"</code> is specified on <code>P</code>,
+then the computed value resolves to (24px,36px).</p>
+</note>
+
+
+<p>If the relative unit is EM (<code>em</code>), then the referenced value is determined as if percentage units
+were used, where <code>1em</code> equals <code>100%</code>.</p>
+
+<note role="example">
+<p>For example, consider a span element <code>S</code>, a child of a paragraph (<code>p</code>) element <code>P</code>.
+If the computed value of <code>tts:fontSize</code> on <code>P</code> is (18px,24px), and if
+<code>tts:fontSize="1em"</code> is specified on <code>S</code>, then this is equivalent to specifying <code>100%</code>,
+which resolves to (24px,24px). However, if <code>tts:fontSize="1em 1em"</code> is specified on <code>S</code>,
+then this is equivalent to specifying <code>100% 100%</code>  which resolves to (18px,24px).</p>
+</note>
+
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+
+
 <p>For the purpose of determining applicability of this style property,
 each character child of a <el>p</el> element is considered to be enclosed in an anonymous
 span.</p>
-<p>If a computed value of the property associated with this attribute is not supported,
-then a presentation processor must use the closest supported value.</p>
+
 <note role="elaboration">
 <p>In this context, the phrase <emph>closest supported value</emph> means the value for which the Euclidean distance between
-the computed font size and the supported font size is minimized. If there are multiple closest supported values equally distant from
-the computed value, then the value most distant from 0 (single length specification) or [0,0] (two length specifications) is used,
-i.e., the largest font size, is used.</p>
-</note>
-<note role="elaboration">
-<p>The expression <code>1c</code> means one cell, where <code>'c'</code> expresses
-the <emph>cell</emph> length unit as defined by <specref ref="style-value-length"/>.
-When a single &lt;length&gt; is expressed using cell units, then it refers to the height of
-the <emph>Computed Cell Size</emph>.
-When two &lt;length&gt; values are expressed using cell units, then the first refers to the width of
-the <emph>Computed Cell Size</emph>, and the second refers to the height of the
-<emph>Computed Cell Size</emph>.</p>
+the computed font size and the supported font size is minimized. If there are multiple closest supported values equally distant from		
+the computed value, then the value most distant from 0 (single length specification) or [0,0] (two length specifications) is 
+used, i.e., the largest font size, is used.</p>		
 </note>
+
+<p>If a computed value of the property associated with this attribute is not supported,
```

Comment:
This paragraph needs to go before the preceding note "In this context, the closest supported value ...", which purpose is to elaborate this paragraph (that now follows the note).